pkg/commands/user.go

```diff
@@ -30,7 +30,7 @@ import (
 
 // for testing
 var (
-	Lookup = util.Lookup
```

Comment:
This seems to be unused by tests? `user_test.go` calls `util.LookupUser` instead, unless I'm misreading.

If this is the case, we can just have code in this package call `user.LookupUser` directly instead of using this var, or maybe it should be using `util.LookupUser`.

Reply:
Excellent! Thanks for reading this code closer than I did. 😅